### ISSUE TITLE
thumbnails correctly sized

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-VIZLAB/hurricane-irma#110]
@mwernimont 

Can you resize the thumbnails for the appropriate platform? I Slack messaged you the actual files.

## Facebook

1560 width
820 height

## Twitter

560 width
300 height

## VIZLAB Landing page

400 width
400 height

## Instagram

1080 width
1080 height

**Comment by lindsayplatt** [comment]
Closed by #128

**Comment by lindsayplatt** [comment]
ImageMagik cmd line code for all three thumbnails (assuming screenshot of the whole fig is called `thumb-base.png`:

```
convert thumb-base.png -thumbnail 400x400^ -extent 400x400 -unsharp 1.5x1+1.5+0 thumb-landing.png

convert thumb-base.png -thumbnail 560x300^ -gravity west -extent 560x300-0-35 -unsharp 1.5x1+1.5+0 thumb-twitter.png

convert thumb-base.png -thumbnail 1200x630^ -extent 1200x630-0+90 -unsharp 1.5x1+1.5+0 thumb-facebook.png
```
